mypy/stubgen.py

```diff
@@ -624,26 +624,25 @@ def visit_func_def(self, o: FuncDef, is_abstract: bool = False,
             # name their 0th argument other than self/cls
             is_self_arg = i == 0 and name == 'self'
             is_cls_arg = i == 0 and name == 'cls'
-            if (annotated_type is None
-                    and not arg_.initializer
-                    and not is_self_arg
-                    and not is_cls_arg):
-                self.add_typing_import("Any")
-                annotation = ": {}".format(self.typing_name("Any"))
-            elif annotated_type and not is_self_arg and not is_cls_arg:
-                annotation = ": {}".format(self.print_annotation(annotated_type))
-            else:
-                annotation = ""
+            annotation = ""
+            if annotated_type and not is_self_arg and not is_cls_arg:
+                print(annotated_type, get_proper_type(annotated_type))
```

Comment:
remove
```suggestion
```

Reply:
Oops, now the whole world can see that I use printf debugging. :P

Reply:
I think everyone does it :)